postgres/datadog_checks/postgres/config.py

```diff
@@ -17,10 +17,26 @@ class PostgresConfig:
 
     def __init__(self, instance):
         self.host = instance.get('host', '')
+        if not self.host:
+            raise ConfigurationError('Specify a Postgres host to connect to.')
         self.port = instance.get('port', '')
         if self.port != '':
             self.port = int(self.port)
+        self.user = instance.get('username', '')
+        if not self.user:
+            raise ConfigurationError('Please specify a user to connect to Postgres.')
+        self.password = instance.get('password', '')
         self.dbname = instance.get('dbname', 'postgres')
+
+        self.application_name = instance.get('application_name', 'datadog-agent')
+        if PY3 and not self.application_name.isascii():
+            raise ConfigurationError("Application name can include only ASCII characters: %s", self.application_name)
+        elif PY2:
+            try:
+                self.application_name.encode('ascii')
+            except UnicodeEncodeError:
+                raise ConfigurationError("Application name can include only ASCII characters: %s", self.application_name)
```

Comment:
CI fails because flake8 finds this line is too long: https://dev.azure.com/datadoghq/integrations-core/_build/results?buildId=23350&view=logs&j=910f82a4-622d-5ace-f877-8b75c045486c&t=b8a76c45-01d6-50b1-e11b-d9fedb0dbe38&l=710

Once above comment is addressed things should be resolved (but can run a `ddev test -fs` just in case…).